routers/api/v1/convert/convert.go

```diff
@@ -220,11 +220,12 @@ func ToOrganization(org *models.User) *api.Organization {
 // ToTeam convert models.Team to api.Team
 func ToTeam(team *models.Team) *api.Team {
 	return &api.Team{
-		ID:          team.ID,
-		Name:        team.Name,
-		Description: team.Description,
-		Permission:  team.Authorize.String(),
-		Units:       team.GetUnitNames(),
+		ID:                      team.ID,
+		Name:                    team.Name,
+		Description:             team.Description,
+		IncludesAllRepositories: team.IncludesAllRepositories,
```

Comment:
Is this compatible with github v3 API?

Reply:
I don't understand the relationship with the github API.
Can you please specify what is it?

Reply:
@lunny github does not have such feature